src/test/scala/ai/eto/rikai/sql/spark/parser/RikaiSparkSQLParserTest.scala

```diff
@@ -39,10 +39,12 @@ class RikaiSparkSQLParserTest extends AnyFunSuite with SparkTestSession {
   }
 
   test("Test parse ML_PREDICT with catalog") {
-    spark.sql("CREATE MODEL foo USING 'test://host/foo'").show()
+    spark.udf.register("bar", (s: Int) => s + 2)
+
+    spark.sql("CREATE MODEL bar USING 'test://host/bar'").show()
 
     val scores =
-      spark.sql("SELECT id, ML_PREDICT('test://host/foo', id) AS score FROM df")
+      spark.sql("SELECT id, ML_PREDICT(bar, id) AS score FROM df")
```

Comment:
why wouldn't `bar` be quoted here?

Reply:
it was a mistake by copying the other tests and forgot to change to a reference to an existing model instead of a temporary model referred by string.